cookbook/core/control_flow/checkpoint.py

```diff
@@ -1,47 +1,48 @@
 """
 Intratask Checkpoints
-------------
+----------------------
 
 .. note::
 
-  This feature is available since flytekit version 0.30.0b6+ and needs a Flyte backend version of atleast 0.19.0+
+  This feature is available from Flytekit version 0.30.0b6+ and needs a Flyte backend version of atleast 0.19.0+.
 
-A checkpoint allows a task to recover from a previous failure by recording the state of a task before the failure and
+A checkpoint recovers a task from a previous failure by recording the state of a task before the failure and
 resuming from the latest recorded state.
 
 Why Intra-task checkpoints?
```

Comment:
```suggestion
Why Intra-task Checkpoints? 
```

Please use https://titlecaseconverter.com/ for capitalizing the headings. Select "Chicago" style.